cordova-js-src/splashscreen.js

```diff
@@ -30,7 +30,8 @@ var schema = (isHosted || isWin10UWP && isMsAppxWeb) ? 'ms-appx-web' : 'ms-appx'
 var fileName = isPhone ? 'splashscreenphone.png' : 'splashscreen.png';
 var splashImageSrc = schema + ':///images/' + fileName;
 
-var splashElement = null, extendedSplashImage = null, extendedSplashProgress = null;
+var splashElement = null, extendedSplashImage = null, extendedSplashProgress = null,
```

Comment:
nit: please put each definition on its own line - IMO it's be much easier to read